### DISCUSSION TITLE
Workshop announcement: A chance to shape the future of TOSS

### DISCUSSION THREAD

**Discussion by russelltrow** [Green-Software-Foundation/ouse#21] [Announcements]
Are you passionate about sustainable software and part of a Green Software Foundation member organization? We're seeking enthusiastic and dedicated practitioners like you to make a meaningful contribution to accelerating the Transforming Organisational Software Sustainability (TOSS) project.

This opportunity is exclusive to members of the Green Software Foundation [Check if your organisation is a member](https://greensoftware.foundation/team).

## The Opportunity
We are looking for people to participate in a series of virtual workshops to shape the future of our [Transforming Organisational Software Sustainability (TOSS) project](https://github.com/Green-Software-Foundation/toss). 

The TOSS project team is actively developing a framework that encapsulates an underlying methodology and decision trees for organisations to 1) Embed 2) Transform and 3) Use software emissions data to help achieve their Net Zero targets. The framework provides companies with an approach to build a future-proof foundation for continuous improvement and change management regarding the sustainable development and use of their software operations.

![Screenshot 2024-06-20 at 11 09 57](https://github.com/Green-Software-Foundation/toss/assets/11027021/06b28a48-a628-4b3f-b424-a83d253433d9)

We are now at the stage where we need help and insights to further develop the decision trees for each of the pillars based on broader contributions from those individuals that recognise the problem that is being addressed.

## We're seeking individuals who have
* A passion for sustainable software
* Familiarity with processes in IT functions in organisations and want to help drive a robust approach for incorporating Green Software processes into IT teams
* Background in the management of technology - Architects, Change Experts, Project / Programme Managers, Procurement Specialists, Compliance Officers, Product Managers, Business Managers
* Ability to commit 1 hour of their time for 6 weeks

## We need you to help us
Work on the decision trees for the 4 pillars - Strategy, Implementation, Operational, Compliance and Regulations

## Workshop Details
The sessions are planned:

* Monday 15th July - TOSS Orientation
* Monday 22nd July - Strategy pillar
* Monday 29th July - Implementation pillar
* Monday 5th August - Operational pillar
* Monday 12th August - Compliance and Regulations pillar
* Monday 19th August - Wrap-up

Each workshop will run between 14:00 - 15:00 UTC.

## Register your interest
To get involved in the workshops simply post a comment below and we will add you to the mailing list.

cc: @PindyBhullar @sk16-dev @seanmcilroy29

**Comment by seppmaier99** [comment]
Hi Russell,

I would be interested in participating the workshops,

Best regards,

Clemens Kunwald

Ps. I belong to AUSY Technologies, Thomas Frers (in the mean time Randstad Digital)

**Reply by russelltrow** [reply]
Hi @seppmaier99, wonderful to have your onboard! I will get you added to the mailing list and provide further details next week.

**Comment by Strangetex** [comment]
Founder of green and responsible tech startup.  University of Athens PhD student in STS / History and Philosophy of Science.    Interested in joining, thanks!

**Reply by russelltrow** [reply]
Hi @Strangetex are you currently associated with a [GSF member organisation](https://greensoftware.foundation/team/)?

This workshop is exclusive to members of the Green Software Foundation.

**Comment by gbertello** [comment]
Hello,

I work in Amadeus (already a GSF member organisation), and I am interested in joining as well.
My tech background is available on LinkedIn:
https://www.linkedin.com/in/guillaume-bertello-067713151/

Let me know if there is an available seat for me!

Thanks
Guillaume

**Reply by russelltrow** [reply]
Hey @gbertello, wonderful to have your onboard! I will get you added to the mailing list and provide further details next week.

**Reply by gbertello** [reply]
Thanks. I have just updated my email address in GitHub. Please use the one on the domain @gmail.com so that I can receive the information accordingly.

Looking forward to attending this workshop with you all!

**Comment by Strangetex** [comment]
Just joined LFCA but not sure if that counts?  sorry if not.

best,
Tracey

**Reply by russelltrow** [reply]
Hi @Strangetex, if you are associated with the LFCA then you simply need to register with the GSF and I can get you added to the list:

https://wiki.greensoftware.foundation/orientation/signup

**Comment by danielvaughan** [comment]
I am interested in being added

**Reply by russelltrow** [reply]
Hey @danielvaughan, wonderful to have your onboard! I will get you added to the mailing list and provide further details this week.

**Comment by florent-morel** [comment]
Hi Russell, 

This is definitely a project I'd like to be part of (even if I won't be able to join most workshops), thanks for setting this up.
As we're building/adapting GSF tools, our current challenge is about adoption of Green Software principles and how to embark organizations in the journey.

**Reply by russelltrow** [reply]
Hey @florent-morel, wonderful to have your onboard! I will get you added to the mailing list and provide further details this week.

**Comment by rvergallo** [comment]
Hi Russell,
I'm a software engineer and researcher at University of Salento (Italy), member of GSF.
I'm interested to join the workshop.
Cheers,
Roberto

**Reply by russelltrow** [reply]
Hey @rvergallo, wonderful to have your onboard! I will get you added to the mailing list and provide further details this week.

**Comment by philippqf** [comment]
Interested. Count me in.

**Reply by russelltrow** [reply]
Hey @philippqf , wonderful to have your onboard! I will get you added to the mailing list and provide further details this week.

**Comment by srini1978** [comment]
I an interested. Please count me in as well. @russelltrow

**Reply by russelltrow** [reply]
Hey @srini1978, wonderful to have your onboard! I will get you added to the mailing list and provide further details this week.

**Comment by hectortosa** [comment]
Hi, Héctor here form Thoughtworks, I would like to be part of the workshop.

Thanks!

**Reply by russelltrow** [reply]
Hey @hectortosa , wonderful to have your onboard! I will get you added to the mailing list and provide further details this week.

**Comment by fullo** [comment]
I'm an independent green software advocate and GSF champion. Can I participate?

**Reply by russelltrow** [reply]
Hey @fullo, thank you so much for your interest in the workshop. It is only open to members of the Green Software Foundation [Check if your organisation is a member](https://greensoftware.foundation/team).

**Comment by Dan-D-Fitt** [comment]
Hi, I would be interested in joining this. I am a software engineer for a an international HVAC manufacturer looking after BIM.  Within the organization, I am also part of the Green working groups they have set up, so this workshop would be of interest to me.

Thank you

**Reply by russelltrow** [reply]
Hey @Dan-D-Fitt , thank you so much for your interest in the workshop. It is only open to members of the Green Software Foundation [Check if your organisation is a member](https://greensoftware.foundation/team).

**Comment by savvy-orora** [comment]
Hello cc: @PindyBhullar 

I'm interested in joining the workshop. It is wonderful working with you on the SCI algorithm publication at Credit Suisse/UBS.

Cheers,
Savitha Sridharan

**Reply by russelltrow** [reply]
Hi @savvy-orora, please take a moment to register with the GSF and I can get you added to the list:

https://wiki.greensoftware.foundation/orientation/signup

**Comment by Dan-D-Fitt** [comment]
Hi,

Does this have to be at the organisational level or at the practitioner (my) level?

My organisation are not a member, so I have registered at myself with you: Would that be sufficient?

The work I am doing, in the role I have just taken over, involves looking after Building Information Modelling (BIM) models and setting up a Green IT Group for my employers, FlaktGroup, who are a global manufacturer of ventilation and HVAC products.

As an IT professional with over 30 years experience, I am in the unique position of being able to help address climate change by using these BIM models to help our customers (and market) select the product with the least impact on the environment and to reduce our internal IT carbon footprint at the same time. As someone who works with software development teams and being a Green champion for the wider business for non IT issues, being part of this workshop would be beneficial from your perspective to give you an insight into how things work within industry.

BIM incidentally is about creating digital 3D twins of our products for use in Architects' software in the design and management of buildings, including using sustainability information to help our industry address the challenges of climate change through better building design.

Ignore our BIM webpage at the moment as that's being updated.

Not many people are in the position I am, where I can make real change, by helping the environment both professionally and privately by having satisfaction for doing it.

Kind regards



Dan Fitt B.Sc (Hon) CITP MBCS
Technical Software Engineer
"Software Development Team"
Presenting tomorrow, today.

FläktGroup

Email: ***@***.******@***.***>
Address: Axial Way, Colchester, Essex, CO4 5ZD

**Comment by SaraMatino** [comment]
Hi Russell, I am interested in the workshop. 

Best regards, 
Sara Matino.

**Reply by russelltrow** [reply]
Hey @SaraMatino, thank you so much for your interest in the workshop. It is only open to members of the Green Software Foundation, [Check if your organisation is a member](https://greensoftware.foundation/team).

**Comment by pdushkin** [comment]
Hi Russell,

Please count me in - I am interested.

Thank you,
Peter

**Reply by russelltrow** [reply]
Hey @pdushkin, wonderful to have you onboard! I will get you added to the mailing list and provide further details this week.

**Comment by llhardy1A** [comment]
Hi @russelltrow ,
I'm interested to be be part of, note that I'll be off for the last date.
Thank you

**Reply by russelltrow** [reply]
Hey @llhardy1A, wonderful to have you onboard! I will get you added to the mailing list and provide further details this week.

**Comment by mbitencourt** [comment]
Hi @russelltrow 
I am interested in contributing and being part of TOSS.
Thank you
Mauricio Bitencourt

**Reply by russelltrow** [reply]
Hey @mbitencourt , wonderful to have your onboard! I will get you added to the mailing list and provide further details this week.

**Comment by lsreenivas** [comment]
Hello  @PindyBhullar  @sk16-dev  @seanmcilroy29 

I am very much interested in the workshop. I am interested in contributing and being part of TOSS..  Thanks

Sreenivasan L
Amadeus Software Labs Bangalore India

**Reply by russelltrow** [reply]
Hi @lsreenivas, please take a moment to register with the GSF and I can get you added to the list:

https://wiki.greensoftware.foundation/orientation/signup

**Comment by lsreenivas** [comment]
Hi Russel,
Thank you. Yes, I have registered with GSF now. Thanks..!
Regards,Sreenivasan L

    On Thursday, July 11, 2024 at 04:11:15 PM GMT+5:30, Russell Trow ***@***.***> wrote:  
 
 


Hi @lsreenivas, please take a moment to register with the GSF and I can get you added to the list:

https://wiki.greensoftware.foundation/orientation/signup

**Comment by Ailane** [comment]
Hi, can you please add me as well, thank you in advance :)

**Comment by CelineJo** [comment]
hello, I would love to be part of it. Some of my colleagues from Amadeus are in it already, but due to vacation period we are making sure there is always one of us in it. Thanks!

**Reply by russelltrow** [reply]
Hi @CelineJo, please take a moment to register with the GSF and I can get you added to the list:

https://wiki.greensoftware.foundation/orientation/signup

**Comment by ghost** [comment]
Hi @PindyBhullar @seanmcilroy29 - As discussed, please could you add me to the Wrap Up summary meeting on 19th August ? Thank you.

**Reply by russelltrow** [reply]
Hi @ParulJalota I've added you to the mailing list and the calendar invites.

**Comment by robin-castellon** [comment]
Hello, hopefully it's not too late for a registration ?
I have colleagues from Amadeus who are registered already, but due to vacation period we are making sure there is always one of us in it. 
I created today a GSF profile. 
Thanks!

**Reply by russelltrow** [reply]
Hey @robin-castellon, wonderful to have your onboard! 

I've added you to the mailing list/calendar invite and will send you the workshop information via email now.

**Comment by russelltrow** [comment]
A huge thank you to everyone who has been involved.
